modules/gapi/src/executor/gstreamingexecutor.cpp

```diff
@@ -492,9 +500,14 @@ void emitterActorThread(std::shared_ptr<cv::gimpl::GIslandEmitter> emitter,
         return;
     }
 
+    GAPI_ITT_STATIC_LOCAL_HANDLE(emitter_hndl, "emitter_region");
+    GAPI_ITT_STATIC_LOCAL_HANDLE(emitter_pull_hndl, "emitter_pull_region");
+
     // Now start emitting the data from the source to the pipeline.
     while (true)
     {
+        GAPI_ITT_AUTO_TRACE_GUARD(emitter_hndl);
```

Comment:
This trace seems also extra

Reply:
Yes

Reply:
But please keep it as the overall `emitter` task. I'd highlight two regions inside it - the one where you `pull` data (you do have this trace already), and the one where you push this data to queue (under `if (result)`)

Reply:
Thanks a lot, done!